_data/changelogs/component-summary-box-accessibility.yml

```diff
@@ -0,0 +1,8 @@
+title: Summary box accessibility tests
+type: component
+items:
+  - date: NNNN-NN-NN
+    summary: Added accessibility tests page.
+    affectsGuidance: true
+    githubPr: 2903
```

Comment:
**issue:** This should point to this PR

```suggestion
    githubPr: 3016
```

Reply:
Resolved in [924bdc1](https://github.com/uswds/uswds-site/pull/3016/commits/924bdc18c76c5cc710c5ee47d88bc13761ff66e4)